che-start-workspace/body.json

```diff
@@ -1,75 +0,0 @@
-{
```

Comment:
Why is it better to have this json included in osioperf.py instead of in separate file?

Reply:
Hi, this is because the osioperf.py is copied to the locust slave nodes via SSH where it is run by the slave agent (this is how locust works in distributed mode). So all the files that the python loads need to be accessible and copied as well.

By including the body.json file's content into a string variable inside python instead of a separate file the only file transferred to slaves is the actual locust file.